configs/pix2pix/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-[ALGORITHM]
```

Comment:
It seems that the tag for `PGGAN` is missed.

Reply:
Ok, I will check.